test/wdio.conf.js

```diff
@@ -31,6 +31,89 @@ async function capabilityDefaults() {
   };
 }
 
+class GitHubStatus {
+  /**
+   * @param {Object} p
+   * @param {string} p.sha
+   * @param {string} p.repo
+   * @param {string} p.token
+   * @param {string} p.context
+   */
+  constructor({sha, repo, token, context}) {
+    this.sha = sha;
+    this.repo = repo;
+    this.token = token;
+    this.context = context;
+  }
+
+  setTarget(priority, target) {
+    if (this.targetPriority > priority) {
+      return;
+    }
+
+    this.targetPriority = priority;
+    this.target = target;
+
+    if (this.lastStatus) {
+      this._send(this.lastStatus);
+    } else {
+      this.update({
+        state: 'pending'
+      })
+    }
+  }
+
+  update(status) {
+    this.lastStatus = status;
+    this._send(status);
+  }
```

Comment:
Maybe?

```suggestion
    if !(this.lastStatus) {
      this.lastStatus = { state: 'pending' }
    }
    
    this._send(this.lastStatus)
  }
```

Reply:
How about this: fafb219087991de95eb877c7707ddad479863fd5

Reply:
Bueno.